articles/quickstart/backend/python/index.yml

```diff
@@ -32,12 +32,12 @@ sdk:
   name: auth0-python
   url: https://github.com/auth0/auth0-python
   logo: python
+hidden_articles:
+  - interactive
 requirements:
-  - python 2.7, 3.3 and up
-  - flask 0.11.1
-  - python-jose-cryptodome 1.3.2
-  - flask-cors 3.0.2
-  - six 1.10.0
+  - Python 3.6 and up
+  - Flask 2.0.3
```

Comment:
I think most readers would know better, but should we consider being slightly less specific about the version listed as required, to avoid confusion that a very specific version is needed for some reason? For example, when Flask 2.0.4 or 2.1 inevitably comes down the road, might our QS saying 2.0.3 specifically be confusing? Maybe just saying "Flask 2.0" and "Authlib 1.0" would be enough here 🤔 Up to you!

Reply:
Yeah - good point 👍